### PR TITLE
RTEMIS-154: Demo feedback fixes.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-154

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Modified School Account issues.
- Added fullscreen progress on ajax calls.

### CODE REVIEW

File: modules/rte_mis_school/rte_mis_school.links.task.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 rte_mis_school.school_registration.edit:
   route_name: rte_mis_school.school_registration.edit
   base_route: entity.user.canonical
-  title: Edit Profile
+  title: Register
```
Comment (jaswinsingh) [inline modules/rte_mis_school/rte_mis_school.links.task.yml:5]: Can you confirm with Madhu, if school is registering for the first time it should be register or something more appropriate.
CC - @kunal-sin

File: modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php [hunk 2]
```diff
@@ -267,6 +267,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       $form['field_siblings_details']['widget']['add_more']['add_more_button_siblings']['#ajax']['callback'] = '::multiEventAjaxWrapper';
       $form['field_siblings_details']['widget']['add_more']['add_more_button_siblings']['#ajax']['wrapper'] = 'school-detail-wrapper';
       $form['field_siblings_details']['widget']['add_more']['add_more_button_siblings']['#validate'] = ['::validateSearchSchool'];
+      $form['field_siblings_details']['widget']['add_more']['add_more_button_siblings']['#ajax']['progress'] = ['type' => 'fullscreen'];
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php:270]: Ok.

File: modules/rte_mis_school/rte_mis_school.module [hunk 3]
```diff
@@ -925,7 +929,7 @@ function rte_mis_school_field_widget_complete_paragraphs_table_widget_form_alter
     $field_widget_complete_form['#attributes']['id'][] = 'fee-details-wrapper';
     // Ajax should called on add more item.
     $field_widget_complete_form['widget']['add_more']['add_more_button_fee_details']['#ajax']['callback'] = 'rte_mis_school_fee_details_add_more_callback';
-
+    $field_widget_complete_form['widget']['add_more']['add_more_button_fee_details']['#ajax']['progress'] = ['type' => 'fullscreen'];
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_school/rte_mis_school.module:932]: ok

File: modules/rte_mis_school/rte_mis_school.module [hunk 4]
```diff
@@ -908,13 +910,15 @@ function rte_mis_school_field_widget_complete_paragraphs_table_widget_form_alter
...
     $field_widget_complete_form['widget']['add_more']['add_more_button_education_level']['#ajax']['callback'] = 'rte_mis_school_education_level_wrapper_callback';
+    $field_widget_complete_form['widget']['add_more']['add_more_button_education_level']['#ajax']['progress'] = ['type' => 'fullscreen'];
     // Add validation for education level paragraph.
     $field_widget_complete_form['widget']['#element_validate'][] = 'rte_mis_school_validate_education_level_paragraph';
   }
   elseif ($context['items']->getName() == 'field_entry_class') {
     $field_widget_complete_form['#attributes']['id'][] = 'entry-class-wrapper';
     // Ajax should called on add more item.
     $field_widget_complete_form['widget']['add_more']['add_more_button_entry_class']['#ajax']['callback'] = 'rte_mis_school_entry_class_wrapper';
+    $field_widget_complete_form['widget']['add_more']['add_more_button_entry_class']['#ajax']['progress'] = ['type' => 'fullscreen'];
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_school/rte_mis_school.module:921]: ok

[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_school/rte_mis_school.module:921]: ok

File: modules/rte_mis_school/rte_mis_school.module [hunk 5]
```diff
@@ -908,13 +910,15 @@ function rte_mis_school_field_widget_complete_paragraphs_table_widget_form_alter
     $field_widget_complete_form['#attributes']['id'][] = 'education-level-wrapper';
     // Ajax should called on add more item.
     $field_widget_complete_form['widget']['add_more']['add_more_button_education_level']['#ajax']['callback'] = 'rte_mis_school_education_level_wrapper_callback';
+    $field_widget_complete_form['widget']['add_more']['add_more_button_education_level']['#ajax']['progress'] = ['type' => 'fullscreen'];
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_school/rte_mis_school.module:913]: ok

File: modules/rte_mis_mail/src/Element/RteMobileNumber.php [hunk 6]
```diff
@@ -66,6 +66,7 @@ public function rteMobileNumberProcess(array $element, FormStateInterface $form_
     // Update the ajax call function and add the logic to show the timer &
     // disable the button.
     $element['send_verification']['#ajax']['callback'] = 'Drupal\rte_mis_mail\Element\RteMobileNumber::verifyAjax';
+    $element['send_verification']['#ajax']['progress'] = ['type' => 'fullscreen'];
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_mail/src/Element/RteMobileNumber.php:69]: Sure

File: modules/rte_mis_school/rte_mis_school.links.task.yml (hunk 1 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_school/rte_mis_school.links.task.yml:5]: As per the feedback list, it is 'Register'. Again it's appropriate to confirm this from Madhu.

**Pull request merged**